src/main/java/com/avispl/symphony/dal/communicator/nec/multisync/SocketCommunicator.java

```diff
@@ -302,7 +305,7 @@ public static String getHexByteString(CharSequence prefix, CharSequence separato
             this.status.setConnectionState(ConnectionState.Unknown);
             this.destroyChannel();
             if (retryOnError) {
-                return this.send(data, true);
+                return this.send(data, false);
```

Comment:
The SocketCommunicator is used in several adapters, and was also added to DAL commons. Is that bug exist in all of them? If so - needs to be fixed there as well!

Reply:
The same code definitely exist in all the other places where SocketCommunicator exists (before it was added to DAL, it was copied in several different projects). I've fixed one in symphony-dal-commons, please review all other adapters.